tests/block_tools.py

```diff
@@ -153,36 +155,49 @@ def __init__(
 
         self.root_path = root_path
         self.local_keychain = keychain
+        self.keychain_proxy: Optional[KeychainProxy] = None
         self._block_time_residual = 0.0
-
-        create_default_chia_config(root_path)
-        create_all_ssl(
-            root_path,
-            private_ca_crt_and_key=get_next_private_ca_cert_and_key(),
-            node_certs_and_keys=get_next_nodes_certs_and_keys(),
-        )
-
         self.local_sk_cache: Dict[bytes32, Tuple[PrivateKey, Any]] = {}
+        self.automated_testing = automated_testing
+
+        if automated_testing:
+            create_default_chia_config(root_path)
+            create_all_ssl(
+                root_path,
+                private_ca_crt_and_key=get_next_private_ca_cert_and_key(),
+                node_certs_and_keys=get_next_nodes_certs_and_keys(),
+            )
         self._config = load_config(self.root_path, "config.yaml")
-        self._config["logging"]["log_stdout"] = True
-        self._config["selected_network"] = "testnet0"
-        for service in ["harvester", "farmer", "full_node", "wallet", "introducer", "timelord", "pool"]:
-            self._config[service]["selected_network"] = "testnet0"
+        if automated_testing:
+            self._config["logging"]["log_stdout"] = True
+            self._config["selected_network"] = "testnet0"
+            for service in [
+                "harvester",
+                "farmer",
+                "full_node",
+                "wallet",
+                "introducer",
+                "timelord",
+                "pool",
+                "simulator",
+            ]:
+                self._config[service]["selected_network"] = "testnet0"
+
+            # some tests start the daemon, make sure it's on a free port
+            self._config["daemon_port"] = find_available_listen_port("BlockTools daemon")
```

Comment:
I think these configs can be passed in using `config_overrides` and this would make the code cleaner

Reply:
Ill move those too

Reply:
This can not be passed in through config_overrides, due to some of the changes being made on the base level of the config file

Reply:
Why can root level changes not work?